Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -1246,9 +1291,14 @@ script:
     arguments:
     - name: resource
       required: true
-      description: A URL for which you want to retrieve the most recent report. You
-        may also specify a scan_id (sha256-timestamp as returned by the URL submission
-        API) to access a specific report
+      description: A URL (single or multiple separated by a comma) for which you want
+        to retrieve the most recent report. You may also specify a scan_id (sha256-timestamp
+        as returned by the URL submission API) to access a specific report
```

Comment:
I did not understand the part of the scan_id. it sound like you can use it instead of the url argument. is that so?

Reply:
Yeah, that works as well:
![image](https://user-images.githubusercontent.com/20818773/54539842-97fbcc80-499f-11e9-9aa6-4732f2234525.png)